src/test/java/org/fungover/haze/MainTest.java

```diff
@@ -110,14 +125,31 @@ void callExecuteCommandWithLTRIMShouldReturnErrorMessageWhenKeyDoesNotExist() {
         assertThat(Main.executeCommand(database, List.of("LTRIM", "key", "2", "3"), hazeList)).isEqualTo("-The key is not present in the database.\r\n");
     }
 
+    @Test
```

Comment:
Evergreen code.
You can try to exchange for the following one:
@Test
@DisplayName("getInputList Should Return First element in the list")
void getInputListShouldReturnFirstElementInTheList() throws IOException {
    String inputString = "First\nSecond\nThird";
    BufferedReader input = new BufferedReader(new StringReader(inputString));
    List<String> result = Main.getInputList(input);
    assertThat(result.get(0)).isEqualTo("First");
}

Reply:
Thank you for the input, Great work going trough the tests! Main test is now updated and evergreen test is gone